### DISCUSSION TITLE
How to change my github username?

### DISCUSSION THREAD

**Discussion by testing-developer01** [community/community#101785] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Everyone,
I just want to know how can I change the github Username?

**Comment by 247webs** [comment]
Hello,

***Sign in to GitHub***:
Sign in github account

**Go to your Account Settings**
In the top-right corner of the GitHub page, click on your profile picture, and then click on "Settings."
On the left sidebar, click on "Account."

**Change your username**
In the "Account settings" page, under the "Username" section, you'll see your current username.
Click on the "Change username" button.

**New username**
GitHub will ask you to enter your new desired username.
Type in the new username you want and check for its availability.

To confirm the change, you'll need to enter your password.
Click "Change username" After providing your password, click the "Change username" button to confirm.

Thanks

**Reply by testing-developer01** [reply]
Thanks a lot :)

**Comment by keishalove-bread** [comment]
Are these the same steps to use to change your GITHUB username tied to the account that GIT hub uses or only for your profile name?  As I'm trying to change my GITHUB username and not the profile name and it looks like I cannot do it following these steps.  Thanks
